doc/conf.py

```diff
@@ -67,7 +67,10 @@ def _warn_node(self, msg, node, *args, **kwargs):
 
 sphinx.environment.BuildEnvironment.warn_node = _warn_node
 
-if os.environ.get('READTHEDOCS', None) == 'True':
+# on_rtd is whether we are on readthedocs.org, this line of code grabbed from docs.readthedocs.org
```

Comment:
Before merging, we'll need to identify how to verify the merging of the final version of this branch doesn't break Luigi's readthedocs.

Reply:
@hirosassa , how you thought how we can be sure that we don't break RTD when merging this PR?

Reply:
I think it is a little bit difficult because it depends on webhook of this repo.
I confirmed that a local build of documents using Sphinx is succeeded.